CHANGELOG.md

```diff
@@ -10,7 +10,131 @@ tries to adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+- [#2028](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2028) Improvements for multivariate and universal polynomials
+- [#2033](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2033) Optimize `gen(::MPolyRing, ::Int)` for `:deglex` and `:degrevlex`
+- [#2024](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2024) Replace `Val`-dispatch by if-else in `Generic.MPoly` for better performance
+- [#2031](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2031) Clean up `monomial_isless` for `Generic.MPoly`
+
+## [0.44.9] - 2025-03-10
+
+### Added
+
+- [#2020](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2020) Add `is_separable` for univariate polynomials
+- [#2017](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2017) Add `is_positive_entry` and `is_negative_entry` for matrices
+- [#2023](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2023) Add `reverse!` for polynomials
+- [#2023](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2023) Add 2-argument `pow!` method delegating to the existing 3-argument version
+
+### Changed
+
+- [#2022](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2022) Fix some references in the Linear Solving docs
+- [#2013](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2013) Some tweaks to `evaluate` methods
+- [#2019](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2019) Make `MPolyFactor` module more robust
+- [#2018](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2018) More intuitive polynomial evaluation
+- [#2016](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2016) Add some refs between `allow_unicode` and friends
+- [#2011](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2011) Add some missing `check_parent` in MPoly arithmetics
+- [#2007](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2007) Wrap `MPoly` factorization for `UnivPoly`
+
+## [0.44.8] - 2025-02-23
+
+### Added
+
+- [#2001](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2001) Add `minors_with_position` and `minors_iterator_with_position`
+- [#2004](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2004) Add `divides!`
+
+### Changed
+
+- [#1993](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1993) Allow creating `UniversalPolyRing` with variables
+- [#1998](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1998) Fix `is_finite` for rational function fields
+
+## [0.44.7] - 2025-02-17
+
+### Added
+
+### Changed
+
+- [#1999](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1999) Delegate factorization for function fields to `FracField` over polynomial rings
+- [#1999](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1999) Make `rand` for function fields produce reduced fractions
+
+## [0.44.6] - 2025-02-13
+
+### Added
+
+- [#1991](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1991) Add `is_zero(::NCRing)` as an alias of `is_trivial`
+
+### Changed
+
+- [#1994](https://github.com/Nemocas/AbstractAlgebra.jl/pull/1994) Simplify and slightly optimize `vars(::MPolyRingElem)`
```

Comment:
```suggestion
```
debatable, but I would count this as a minor performance improvement / refactoring